sdk/common.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package sdk
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"path/filepath"
+
+	"github.com/edgelesssys/contrast/internal/atls"
+	"github.com/edgelesssys/contrast/internal/attestation/certcache"
+	"github.com/edgelesssys/contrast/internal/attestation/snp"
+	"github.com/edgelesssys/contrast/internal/attestation/tdx"
+	"github.com/edgelesssys/contrast/internal/fsstore"
+	"github.com/edgelesssys/contrast/internal/logger"
+	"github.com/edgelesssys/contrast/internal/manifest"
+)
+
+const cacheDirEnv = "CONTRAST_CACHE_DIR"
+
+// ValidatorsFromManifest returns a list of validators corresponding to the reference values in the given manifest.
+// Originally an unexported function in the contrast CLI.
+// Can be made unexported again, if we decide to move all userapi calls from the CLI to the SDK.
+func ValidatorsFromManifest(m *manifest.Manifest, log *slog.Logger, hostData []byte) ([]atls.Validator, error) {
```

Comment:
`hostData` should be called `coordinatorPolicyChecksum` or something like that.

I also wonder where this would come from. For Contrast releases, we're embedding it during build. But other users of the SDK?

Reply:
changed. see other comment.